lib/people.js

```diff
@@ -0,0 +1,332 @@
+            // simply do
+            mixpanel.people.increment('bob', 'page_views');
+
+            // to decrement a counter, pass a negative number
+            mixpanel.people.increment('bob', 'credits_left', -1);
+
+            // like mixpanel.people.set(), you can increment multiple properties at once:
+            mixpanel.people.increment('bob', {
+                counter1: 1,
+                counter2: 3,
+                counter3: -2
+            });
+    */
+    increment(distinct_id, prop, by, modifiers, callback) {
+        // TODO extract to ProfileHelpers
+
+        var $add = {};
+
+        if (typeof(prop) === 'object') {
+            if (typeof(by) === 'object') {
+                callback = modifiers;
+                modifiers = by;
+            } else {
+                callback = by;
+            }
+            for (const [key, val] of Object.entries(prop)) {
+                if (isNaN(parseFloat(val))) {
+                    if (this.mixpanel.config.debug) {
+                        console.error("Invalid increment value passed to mixpanel.people.increment - must be a number");
+                        console.error("Passed " + key + ":" + val);
+                    }
+                } else {
+                    $add[key] = val;
+                }
+            };
+        } else {
+            if (typeof(by) === 'number' || !by) {
+                by = by || 1;
+                $add[prop] = by;
+                if (typeof(modifiers) === 'function') {
+                    callback = modifiers;
+                }
+            } else if (typeof(by) === 'function') {
+                callback = by;
+                $add[prop] = 1;
+            } else {
+                callback = modifiers;
+                modifiers = (typeof(by) === 'object') ? by : {};
+                $add[prop] = 1;
+            }
+        }
+
+        var data = {
+            '$add': $add,
+            '$token': this.mixpanel.token,
+            '$distinct_id': distinct_id
+        };
+
+        data = merge_modifiers(data, modifiers);
+
+        if (this.mixpanel.config.debug) {
+            console.log("Sending the following data to Mixpanel (Engage):");
+            console.log(data);
+        }
+
+        this.mixpanel.send_request({ method: "GET", endpoint: "/engage", data: data }, callback);
+    }
+
+    /**
+        people.append(distinct_id, prop, value, modifiers, callback)
+        ---
+        Append a value to a list-valued people analytics property.
+
+        usage:
+
+            // append a value to a list, creating it if needed
+            mixpanel.people.append('bob', 'pages_visited', 'homepage');
+
+            // like mixpanel.people.set(), you can append multiple properties at once:
+            mixpanel.people.append('bob', {
+                list1: 'bob',
+                list2: 123
+            });
+    */
+    append(distinct_id, prop, value, modifiers, callback) {
+        // TODO extract to ProfileHelpers
```

Comment:
```suggestion
        // TODO extract to ProfileHelpers once $append is supported by /groups endpoint
```

Reply:
Actually my thinking here was that it's worth extracting these to ProfileHelpers regardless, so that all this logic is centralized there rather than split between ProfileHelpers and MixpanelPeople.